docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -56,9 +58,6 @@ Cause Model Template
       `directive <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#directives>`_
       from :file:`index.rst`.
 
-Disease Description
--------------------
-
 .. todo::
```

Comment:
I think it would be helpful here to add a heading above this todo--perhaps on the same level as "GBD 2017 framework" and "Vivarium framework", as i think @collijk suggested at some point.

Reply:
Oh oops I was viewing this inline, and missed those review comments above--in reply to your aesthetic concern @NathanielBlairStahn , even if expert opinion isn't as immediately relevant to the construction of our model as the gbd/vivarium sections, it's conceptually clearer to me to have that orient-me-to-the-disease/injury under a top-level heading that is distinct from the other two. I think that heading could be either of the flavor 'disease overview' _or_ 'introduction'

Reply:
@beatrixkh , what would be the difference between a 'disease overview' and an 'introduction' in your view?

If we put the expert opinion back in its own section, as you suggest, and we follow @collijk 's suggestion for layout, then there would also be a brief paragraph between the document title and the first section, giving an overview of the entire document.

Reply:
I'm picturing 'disease overview' containing only the expert opinion (or perhaps other general information specific to the disease/cause, though we haven't done anything like that thus far and i don't have examples), and excluding metadata about the page itself.

In the format you and james are suggesting, if we include a table of contents, i was picturing 'introduction' effectively being a broad enough word to be able to hold both a ToC and the disease overview.